test/local.mk

```
@@ -0,0 +1,37 @@
+# Despite the registry running in Cluster, we need to load the container image with `kind load`.
+# There were problems trying to pull container image from registry ("no such host") even though Crossplane could pull the package image...
+.PHONY: package-install
+package-install: CROSSPLANE_REGISTRY = localhost:5000
```

Missing kubeconfig, causing it to fail.

```suggestion
package-install: export KUBECONFIG = $(KIND_KUBECONFIG)
package-install: CROSSPLANE_REGISTRY = localhost:5000
```